languages/rust/exercises/concept/entry-api/.docs/after.md

```diff
@@ -0,0 +1,13 @@
+The entry API is often the most idiomatic way to manipulate the contents of a map structure. It's especially useful when you need to alter multiple entries in a map structure.
+
+This exercise covered one particular pattern that the entry API is commonly used for, namely using `HashMap`s to store the frequency of elements.
+
+Another way this same pattern can be achieved is by using the `and_modify` method in conjunction with either the `or_default` or `or_insert` methods:
+
+```rust
+map.entry(key).and_modify(|v| *v += 1).or_default();
+
+map.entry(key).and_modify(|v| *v += 1).or_insert(0);
+```
```

Comment:
It's pedantic, but I'd like to break this into two distinct code blocks, to reinforce that one shouldn't use both in sequence in the same section of code.